gnmi_server/gnoi.go

```diff
@@ -2,27 +2,59 @@ package gnmi
 
 import (
 	"context"
+	"errors"
+	"os"
 	gnoi_system_pb "github.com/openconfig/gnoi/system"
 	log "github.com/golang/glog"
 	"time"
 	spb "github.com/sonic-net/sonic-gnmi/proto/gnoi"
 	transutil "github.com/sonic-net/sonic-gnmi/transl_utils"
+	io "io/ioutil"
+	ssc "github.com/sonic-net/sonic-gnmi/sonic_service_client"
 	spb_jwt "github.com/sonic-net/sonic-gnmi/proto/gnoi/jwt"
+	"github.com/sonic-net/sonic-gnmi/common_utils"
 	"google.golang.org/grpc/status"
 	"google.golang.org/grpc/codes"
 	"os/user"
 	"encoding/json"
 	jwt "github.com/dgrijalva/jwt-go"
 )
 
+func RebootSystem(fileName string) error {
```

Comment:
>RebootSystem [](http://example.com/codeflow?start=5&length=12)

Is it "reboot" or "config reload"?

Reply:
Use reboot to implement "config reload", and please refer to: https://github.com/openconfig/gnoi/blob/main/system/system.proto#L131